guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -31,9 +28,39 @@ ifdef::satellite[]
 |/var/lib/containers if using {insights-iop} |10 GB |15 GB
 endif::[]
 |====
++
+These values are based on expected use case scenarios and can vary according to individual environments.
+ifdef::katello,satellite[]
+The runtime size was measured with {EL} 7, 8, and 9 repositories synchronized.
+endif::[]
+ifdef::katello,orcharhino,satellite[]
+* If you mount the `/tmp` directory as a separate file system, use the `exec` mount option in the `/etc/fstab` file.
++
+If `/tmp` is already mounted with the `noexec` option, change the option to `exec` and remount the file system.
+This is a requirement for the `puppetserver` service to work.
+* If you mount the `/var/lib/pulp` directory as an NFS share, specify the SELinux context of the `/var/lib/pulp` directory in the file system table.
+Add the following lines to `/etc/fstab`:
++
+[source, none, options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+nfs.example.com:/nfsshare  /var/lib/pulp  nfs  context="system_u:object_r:var_lib_t:s0"  1 2
```

Comment:
`/var/lib/pulp` should be `pulpcore_var_lib_t` these days, not `var_lib_t`.

Reply:
Fixed here and filed https://github.com/theforeman/foreman-documentation/issues/4196 for the earlier versions